src/common/utils.cc

```diff
@@ -50,8 +50,6 @@ Normalize(const DataSet& dataset) {
     auto dim = dataset.GetDim();
     float* data = (float*)dataset.GetTensor();
 
-    LOG_KNOWHERE_INFO_ << "vector normalize, rows " << rows << ", dim " << dim;
```

Comment:
you can change the log level from INFO to DEBUG

Reply:
Done.